docs/release/release-tasks.md

```diff
@@ -120,6 +120,7 @@ This comes down to changing occurrences of the old version to the new version, e
       Please note that both `InitWithKubernetesVersion` and `WorkloadKubernetesVersion` should be the highest mgmt cluster version supported by the respective Cluster API version.
 2. Update `create-local-repository.py` and `tools/internal/tilt-prepare/main.go`: `v1.4.99` => `v1.5.99`.
 3. Make sure all tests are green (also run `pull-cluster-api-e2e-full-main` and `pull-cluster-api-e2e-workload-upgrade-1-23-latest-main`).
+4. Remove unsupported release version of Cluster API. For example remove `v1.3` while working on `v1.6`.
```

Comment:
```suggestion
4. Remove an unsupported release version of Cluster API from the Makefile target that generates e2e templates. For example, remove `v1.3` while working on `v1.6`. 
```

Reply:
@chiukapoor we also need to change notes for CI manager tasks from:

```
Remove tests for old release branches according to our policy documented in [Support and guarantees](https://github.com/chiukapoor/cluster-api/blob/issue-9485/CONTRIBUTING.md#support-and-guarantees) For example, let's assume we just created tests for v1.4, then we can now drop test coverage for the release-1.1 branch.
```

to:

```
Remove tests from the test-infra repository for old release branches according to our policy documented in [Support and guarantees](https://github.com/chiukapoor/cluster-api/blob/issue-9485/CONTRIBUTING.md#support-and-guarantees). For example, let's assume we just created tests for v1.4, then we can now drop test coverage for the release-1.1 branch.
```

Reply:
Inculcating the changes